Next.js 16.2 build failed: Failed to fetch Google Fonts

### Link to the code that reproduces this issue

https://github.com/pawelhoros/next-16.2-google-fonts

### To Reproduce

Pull this repo, then npm run build.

`npm run build

> build
> next build

▲ Next.js 16.2.1-canary.1 (Turbopack)

  Creating an optimized production build ...
Turbopack build encountered 2 warnings:
[next]/internal/font/google/geist_e193ce59.module.css
Error while requesting resource
There was an issue establishing a connection while requesting https://fonts.googleapis.com/css2?family=Geist:wght@100..900&display=swap

Import trace:
  Server Component:
    [next]/internal/font/google/geist_e193ce59.module.css
    [next]/internal/font/google/geist_e193ce59.js
    ./app/layout.tsx


[next]/internal/font/google/geist_mono_b4d4e0d7.module.css
Error while requesting resource
There was an issue establishing a connection while requesting https://fonts.googleapis.com/css2?family=Geist+Mono:wght@100..900&display=swap

Import trace:
  Server Component:
    [next]/internal/font/google/geist_mono_b4d4e0d7.module.css
    [next]/internal/font/google/geist_mono_b4d4e0d7.js
    ./app/layout.tsx



> Build error occurred
Error: Turbopack build failed with 2 errors:
[next]/internal/font/google/geist_e193ce59.module.css
next/font: error:
Failed to fetch `Geist` from Google Fonts.

Import trace:
  Server Component:
    [next]/internal/font/google/geist_e193ce59.module.css
    [next]/internal/font/google/geist_e193ce59.js
    ./app/layout.tsx


[next]/internal/font/google/geist_mono_b4d4e0d7.module.css
next/font: error:
Failed to fetch `Geist Mono` from Google Fonts.

Import trace:
  Server Component:
    [next]/internal/font/google/geist_mono_b4d4e0d7.module.css
    [next]/internal/font/google/geist_mono_b4d4e0d7.js
    ./app/layout.tsx


    at ignore-listed frames`

Switching back to 16.1.7 and everything works.

### Current vs. Expected behavior

Should build with success.

### Provide environment information

```bash
Operating System:
  Platform: darwin
  Arch: arm64
  Version: Darwin Kernel Version 25.3.0: Wed Jan 28 20:56:42 PST 2026; root:xnu-12377.91.3~2/RELEASE_ARM64_T8142
  Available memory (MB): 32768
  Available CPU cores: 10
Binaries:
  Node: 24.14.0
  npm: 11.9.0
  Yarn: N/A
  pnpm: 10.32.1
Relevant Packages:
  next: 16.2.1-canary.1 // Latest available version is detected (16.2.1-canary.1).
  eslint-config-next: N/A
  react: 19.2.4
  react-dom: 19.2.4
  typescript: 5.9.3
Next.js Config:
  output: N/A
```

### Which area(s) are affected? (Select all that apply)

Not sure

### Which stage(s) are affected? (Select all that apply)

next build (local)

### Additional context

_No response_

Same issue, I can't load a page on the dev server.

Module not found: Can't resolve '@vercel/turbopack-next/internal/font/google/cssmodule.module.css'

```
> 1 | ...%22path%22:%22layout.tsx%22,%22import%22:%22Geist_Mono%22,%22arguments%22:[{%22variable%22:%2...
  |    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  2 | ...
  3 | ...
  4 | ...

http2 feature is not enabled

Debug info:
- Execution of <ModuleAssetContext as AssetContext>::resolve_asset failed
- Execution of resolve failed
- Execution of resolve_internal failed
- Execution of NextFontGoogleCssModuleReplacer::import_map_result failed
- Execution of FetchClientConfig::fetch failed
- http2 feature is not enabled

Error while looking up import map: http2 feature is not enabled

Debug info:
- Execution of NextFontGoogleCssModuleReplacer::import_map_result failed
- Execution of FetchClientConfig::fetch failed
- http2 feature is not enabled

Import trace:
  Server Component:
    [next]/internal/font/google/geist_mono_8d43a2aa.js
    ./app/layout.tsx
```

I have the same issue with `next dev` and `next build` on v16.2:

```
> next dev

▲ Next.js 16.2.1 (Turbopack)
- Local:         http://localhost:3000
- Network:       http://xxx.xxx.xxx.xxx:3000
- Environments: .env.local, .env
✓ Ready in 2.8s
- Experiments (use with caution):
  · optimizePackageImports


thread 'tokio-runtime-worker' (19604) panicked at C:\Users\Administrator\.cargo\registry\src\index.crates.io-1949cf8c6b5b557f\hyper-util-0.1.17\src\client\legacy\client.rs:564:33:
http2 feature is not enabled
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
○ Compiling /[locale] ...
⨯ [next]/internal/font/google/inter_fa93af41.js:1:1
Module not found: Can't resolve '@vercel/turbopack-next/internal/font/google/cssmodule.module.css'
> 1 | ...oogle/cssmodule.module.css?{%22path%22:%22layout.tsx%22,%22import%22:%22Inter%22,%22arguments%22:[{%22subsets%22:[%22latin%22]...
    |    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  2 | ...
  3 | ...
  4 | ...

http2 feature is not enabled

Debug info:
- Execution of <ModuleAssetContext as AssetContext>::resolve_asset failed
- Execution of resolve failed
- Execution of resolve_internal failed
- Execution of NextFontGoogleCssModuleReplacer::import_map_result failed
- Execution of FetchClientConfig::fetch failed
- http2 feature is not enabled
Error while looking up import map: http2 feature is not enabled

Debug info:
- Execution of NextFontGoogleCssModuleReplacer::import_map_result failed
- Execution of FetchClientConfig::fetch failed
- http2 feature is not enabled


Import trace:
  Server Component:
    [next]/internal/font/google/inter_fa93af41.js
    ./src/app/layout.tsx

https://nextjs.org/docs/messages/module-not-found

 GET / 500 in 12.0s (next.js: 11.3s, proxy.ts: 655ms, application-code: 121ms)
```

```
> next info

Operating System:
  Platform: win32
  Arch: arm64
  Version: Windows 11 Pro
  Available memory (MB): 15985
  Available CPU cores: 12
Binaries:
  Node: 25.1.0
  npm: 11.6.2
  Yarn: 4.10.3
  pnpm: 10.33.0
Relevant Packages:
  next: 16.2.1
  eslint-config-next: N/A
  react: 19.2.4
  react-dom: 19.2.4
  typescript: 6.0.2
Next.js Config:
  output: N/A
```

Downgrading to 16.1 removed the issue.

Also running into this issue.
I'm running on Windows ARM, and have tried deleting node_modules, .next, pnpm-lock.yaml and reinstalling.

```
thread 'tokio-runtime-worker' (29556) panicked at C:\Users\Administrator\.cargo\registry\src\index.crates.io-1949cf8c6b5b557f\hyper-util-0.1.17\src\client\legacy\client.rs:564:33:
http2 feature is not enabled
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
○ Compiling /login ...
⨯ [next]/internal/font/google/geist_a71539c9.js:1:1
Module not found: Can't resolve '@vercel/turbopack-next/internal/font/google/cssmodule.module.css'
> 1 | ...dule.css?{%22path%22:%22layout.tsx%22,%22import%22:%22Geist%22,%22arguments%22:[{%22variable%22:%22--font-ge...
    |    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  2 | ...
  3 | ...
  4 | ...

http2 feature is not enabled

Debug info:
- Execution of <ModuleAssetContext as AssetContext>::resolve_asset failed
- Execution of resolve failed
- Execution of resolve_internal failed
- Execution of NextFontGoogleCssModuleReplacer::import_map_result failed
- Execution of FetchClientConfig::fetch failed
- http2 feature is not enabled
Error while looking up import map: http2 feature is not enabled

Debug info:
- Execution of NextFontGoogleCssModuleReplacer::import_map_result failed
- Execution of FetchClientConfig::fetch failed
- http2 feature is not enabled


Import trace:
  Server Component:
    [next]/internal/font/google/geist_a71539c9.js
    ./src/app/layout.tsx

https://nextjs.org/docs/messages/module-not-found


⚠ [next]/internal/font/google/geist_mono_8d43a2aa.module.css
Error while requesting resource
There was an issue establishing a connection while requesting https://fonts.googleapis.com/css2?family=Geist+Mono:wght@100..900&display=swap

Import trace:
  Server Component:
    [next]/internal/font/google/geist_mono_8d43a2aa.module.css
    [next]/internal/font/google/geist_mono_8d43a2aa.js
    ./src/app/layout.tsx


⚠ [next]/internal/font/google/geist_mono_8d43a2aa.module.css
next/font: warning:
Failed to download `Geist Mono` from Google Fonts. Using fallback font instead.

Import trace:
  Server Component:
    [next]/internal/font/google/geist_mono_8d43a2aa.module.css
    [next]/internal/font/google/geist_mono_8d43a2aa.js
    ./src/app/layout.tsx

```

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -320,7 +320,7 @@ rand = "0.10.0"
 rayon = "1.10.0"
 regex = "1.10.6"
 regress = "0.10.4"
-reqwest = { version = "0.13.1", default-features = false }
+reqwest = { version = "0.13.2", default-features = false }
 ringmap = "0.2.3"
 roaring = "0.10.10"
 rstest = "0.16.0"

